.github/workflows/glean-probe-scraper.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: Glean probe-scraper
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Comment:
Question: this will only validate data on pull requests, is that right? I'm assuming we'd only want changes to the `main` branch to be published.